frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -76,14 +146,14 @@ const CampCreationPage = (): React.ReactElement => {
       case CampCreationPages.RegistrationFormPage:
         return (
           <RegistrationForm
-            registrationFormDummyOne={registrationFormDummyOne}
-            registrationFormDummyTwo={registrationFormDummyTwo}
-            toggleRegistrationFormDummyOne={() =>
-              setRegistrationFormDummyOne(!registrationFormDummyOne)
-            }
-            toggleRegistrationFormDummyTwo={() =>
-              setRegistrationFormDummyTwo(!registrationFormDummyTwo)
+            formTemplateQuestions={
+              formTemplateQuestions ? formTemplateQuestions.formQuestions : []
```

Comment:
Oh I see what you mean, I rather not change the state though since that would complexify the useEffect of getting the form template from the database, and I'd rather keep that as simple as possible and deal with this conditional check later.